code/datums/diseases/advance/symptoms/heal.dm

```diff
@@ -247,9 +247,7 @@
 		addtimer(CALLBACK(src, .proc/coma, M), 60)
 
 /datum/symptom/heal/coma/proc/coma(mob/living/M)
-	if(deathgasp)
-		M.emote("deathgasp")
-	M.fakedeath("regenerative_coma")
+	M.fakedeath("regenerative_coma", deathgasp)

```

Comment:
the second arg is `silent`, meaning if deathgasp is `TRUE` you want this to be `FALSE`.
https://github.com/tgstation/tgstation/blob/2bfcb644b4ac50064e6838ce8ca379109fc109da/code/modules/mob/living/status_procs.dm#L448

```suggestion
	M.fakedeath("regenerative_coma", !deathgasp)
```